bokun.yaml

```diff
@@ -486,6 +486,7 @@ paths:
         - activity
       operationId: findActivityBySlug
       summary: Look up Activity by slug
+      description: 'Use corresponding slug (human readable string that is used to identify the product) for fetching this product. Slug is set in Product overview -> Actions -> Slugs, then reindexing (republish) of the product is needed.'
```

Comment:
"... then reindexing (republish) of the product is needed."

If manual reindexing is needed after changing slugs, then that is a bug. You should report that to clubhouse.

Reply:
ok, will be done.